kmath-tensors/src/commonTest/kotlin/space/kscience/kmath/tensors/core/TestDoubleAnalyticTensorAlgebra.kt

```diff
@@ -0,0 +1,35 @@
+package space.kscience.kmath.tensors.core
+
+import space.kscience.kmath.tensors.core.algebras.DoubleAnalyticTensorAlgebra
+import kotlin.math.abs
+import kotlin.math.exp
+import kotlin.test.Test
+import kotlin.test.assertTrue
+
+class TestDoubleAnalyticTensorAlgebra {
```

Comment:
Test classes are usually made `internal` to prevent problems if someone imports a test source set.